package.json

```diff
@@ -32,8 +32,8 @@
   "homepage": "https://github.com/Azure/autorest.python#readme",
   "devDependencies": {
     "@actions/github": "6.0.0",
-    "@chronus/chronus": "^1.0.2",
-    "@chronus/github": "^1.0.2",
+    "@chronus/chronus": "^0.12.1",
+    "@chronus/github": "^0.4.4",
```

Comment:
why is this downgraded?

Reply:
same question. @iscai-msft Could you provide more context?